docs/_config.yml

```diff
@@ -7,7 +7,7 @@ locale                   : "en-US"
 title                    : "Hyperspace"
 title_separator          : "-"
 subtitle                 : "An Indexing Subsystem for Apache Spark™"
-url                      : https://microsoft.github.io
+url                      : https://rapoth.github.io
```

Comment:
remove?

Reply:
Oh, just did it. Should reflect now. Good catch!